core/src/bson_type_info.rs

```diff
@@ -80,7 +107,7 @@ impl BsonTypeInfo {
         sql_code: None,
         is_auto_unique_value: Some(false),
         is_unsigned: Some(false),
-        num_prec_radix: Some(10),
+        num_prec_radix: Some(2),
```

Comment:
num_prec_radix is supposed to be 2 for DOUBLES as I read the spec. However, the way we are listing precision here is in number of decimal digits, so I'm not 100% sure on this change, unlike the other

Reply:
The correct answer may actually be to change the precision to be in bits

Reply:
I think 15 for the precision is correct here.
See SQL_DOUBLE in [https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/column-size?view=sql-server-ver16](https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/column-size?view=sql-server-ver16)